CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## `Pending`
+
+* Introduced `init_once` that is an initializer that requires the contract was not initialized yet.
```

Comment:
We may want to document the exact state machine of contract lifetime, i.e. smth along lines of 

```
 Deploy -> First run -> Init once -> Init -> method call
              -> Subsequent runs -> Init -> method call
```